### ISSUE TITLE
Add ability for users to save project tags 

### ISSUE THREAD

**Issue by murphysj** [EarthSystemCoG/COG#1000]
WHO: Cecelia

Say there are 100 tags on the system. A user says these 5 tags are important to me. 

* save as part of the user profile object??
* add a save button to the Project Browser display (along with reset and go)

**Comment by murphysj** [comment]
I added a place holder button and text in the project tag widget to do this. Who knows is C will want that.

**Comment by LucaCinquini** [comment]
I am actually wondering if this is something we need to add later. As we have so very few projects tagged right now, it seems like unnecessary complexity: the user
can find the available tags without much effort in scrolling…

On Feb 27, 2015, at 5:07 PM, Sylvia Murphy <notifications@github.com<mailto:notifications@github.com>> wrote:


I added a place holder button and text in the project tag widget to do this. Who knows is C will want that.

**Comment by murphysj** [comment]
I am not sure it is necessary AT ALL but Cecelia thinks it will become a
big deal. I am happy postponing it until it becomes an issue.

On Sat, Feb 28, 2015 at 4:34 AM, Luca Cinquini <notifications@github.com>
wrote:

> I am actually wondering if this is something we need to add later. As we
> have so very few projects tagged right now, it seems like unnecessary
> complexity: the user
> can find the available tags without much effort in scrolling…
>
> On Feb 27, 2015, at 5:07 PM, Sylvia Murphy <notifications@github.com
> <mailto:notifications@github.com>> wrote:
>
>
> I added a place holder button and text in the project tag widget to do
> this. Who knows is C will want that.
>
> —
> Reply to this email directly or view it on GitHub<
> https://github.com/EarthSystemCoG/COG/issues/1000#issuecomment-76496258>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EarthSystemCoG/COG/issues/1000#issuecomment-76522474>.
>



-- 
******************************************
Sylvia Murphy
NESII/CIRES/NOAA Earth System Research Laboratory
325 Broadway, Boulder CO 80305
Time Zone: U.S. Mountain
Web: http://www.esrl.noaa.gov/nesii/
Email: sylvia.murphy@noaa.gov
Phone: 303-497-7753

**Comment by murphysj** [comment]
Have SAVE button in project widget. Save projects to the user_profile object. Display the tags under a new tab in the PB. 

Under the new tab,

you see the name of the tag as the accordion, with the list of project beneath it, mimicking the This tab.

**Comment by LucaCinquini** [comment]
This ticket took a LONG time because we need to taken federation into account. It is now implemented. The user tags are always saved on the site where the user has created the account (the user 'home' site), and propagated to the site where the user is currently browsing in real time. Whenever a tag is saved or deleted, the "Tags" tab is selected.
Note that I have decided to allow deletion of tags directly from the browser, not from the user profile page, to shortcut the process.
